undertow/src/main/java/org/wildfly/extension/undertow/HostSingleSignOnDefinition.java

```diff
@@ -22,16 +31,21 @@ public class HostSingleSignOnDefinition extends SingleSignOnDefinition {
             .build();
 
     public HostSingleSignOnDefinition() {
-        super();
-        setDeprecated(UndertowSubsystemModel.VERSION_12_0_0.getVersion());
+        super(new BiFunction<>() {
+            @Override
+            public ResourceDefinition.Builder apply(ResourceRegistration registration, ResourceDescriptionResolver resolver) {
+                return ResourceDefinition.builder(registration, resolver, UndertowSubsystemModel.VERSION_12_0_0);
+            }
+        }, new BiFunction<>() {
+            @Override
+            public ResourceDescriptor.Builder apply(ResourceDescriptor.Builder builder, ResourceServiceConfigurator configurator) {
+                return builder.withAddResourceOperationTransformation(AdminOnlyOperationStepHandlerTransformer.INSTANCE);
+            }
+        });
     }
 
     @Override
-    public void registerOperations(ManagementResourceRegistration registration) {
-        ResourceDescriptor descriptor = new ResourceDescriptor(this.getResourceDescriptionResolver())
-                .addAttributes(SingleSignOnDefinition.Attribute.class).addCapabilities(() -> HOST_SSO_CAPABILITY)
```

Comment:
The changes around here make `HOST_SSO_CAPABILITY` never registered and the field `org.wildfly.extension.undertow.HostSingleSignOnDefinition#HOST_SSO_CAPABILITY` never used.

Reply:
Actually - that RuntimeCapability should have been deleted back in WF28 when this resource was deprecated and became admin-only.  Since then it no longer provided its original capability service type.